azure-cs-webserver/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+using Pulumi;
+using Pulumi.Azure.Compute;
+using Pulumi.Azure.Compute.Inputs;
+using Pulumi.Azure.Core;
+using Pulumi.Azure.Network;
+using Pulumi.Azure.Network.Inputs;
+
+class Program
+{
+    static Task<int> Main(string[] args)
+    {
+        return Deployment.RunAsync(() => {
+            var resourceGroup = new ResourceGroup("server-rg");
+
+            var network = new VirtualNetwork("server-network",
+                new VirtualNetworkArgs
+                {
+                    ResourceGroupName = resourceGroup.Name,
+                    AddressSpaces = { "10.0.0.0/16" },
+                    Subnets =
+                    {
+                        new VirtualNetworkSubnetsArgs { Name = "default",  AddressPrefix = "10.0.1.0/24" }
+                    },
```

Comment:
This code yields a warning (as any other `InputList` does): `Object or collection initializer implicitly dereferences possibly null member 'VirtualNetworkArgs.Subnets'`.
Any reason why we mark those lists as nullable? @pgavlin @CyrusNajmabadi